fetch/client/src/main/scala/endpoints4s/fetch/TextEncoder.scala

```diff
@@ -0,0 +1,12 @@
+package endpoints4s.fetch
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSGlobal
+import scala.scalajs.js.typedarray.Uint8Array
+
+//TODO could be used for chunked requests
+@js.native
+@JSGlobal
+class TextEncoder(utfLabel: js.UndefOr[String] = js.undefined) extends js.Object {
```

Comment:
Same here regarding privateness.

Reply:
Will remove, no longer needed because `ChunkedEntities` will not be implemented.

Reply:
Do we keep it, ultimately?